cfgov/ask_cfpb/models/blocks.py

```diff
@@ -0,0 +1,203 @@
+from __future__ import absolute_import
+
+from django.core.exceptions import ObjectDoesNotExist, ValidationError
+from django.forms.utils import ErrorList
+
+from wagtail.wagtailcore import blocks
+
+from ask_cfpb.models.django import Answer
+from v1 import blocks as v1_blocks
```

Comment:
Would it make sense to import the specific blocks you need here, since there are only a few of them?  e.g. `from v1.blocks import HeadingLevelBlock, ... `